package proto_ff;

import "yd_fieldoptions.proto";


message functionunlockfunctionUnlock
{
	optional int32 functionId = 1[(yd_fieldoptions.field_cname) = "功能id"];
	optional int32 openDaily = 2[(yd_fieldoptions.field_cname) = "开服多少天开放"];
	optional int32 openType = 3[(yd_fieldoptions.field_cname) = "开放方式"];
	optional int64 openVal = 4[(yd_fieldoptions.field_cname) = "开放数值"];
	optional int32 iconOpenVal = 5[(yd_fieldoptions.field_cname) = "提前显示icon参数"];
	optional int32 belong = 6[(yd_fieldoptions.field_cname) = "附属"];
}


message Sheet_functionunlockfunctionUnlock
{
	repeated functionunlockfunctionUnlock functionunlockfunctionUnlock_List = 1[(yd_fieldoptions.field_arysize)=357];
}
